src/component/container/pattern_list.tsx

```diff
@@ -66,20 +71,14 @@ export class PatternListContainer extends React.Component<PatternListContainerPr
 		);
 	}
 
-	public createItemsFromFolder(folder: PatternFolder): PatternListContainerItemProps[] {
+	public createItemsFromStyleguide(styleguide: Styleguide): PatternListContainerItemProps[] {
 		const result: PatternListContainerItemProps[] = [];
-		if (folder) {
-			folder.getChildren().forEach((child: PatternFolder) => {
-				const childItem: PatternListContainerItemProps = { value: child.getName() };
-				childItem.children = this.createItemsFromFolder(child);
-				result.push(childItem);
-			});
-
-			folder.getPatterns().forEach((pattern: Pattern) => {
+		if (styleguide) {
```

Comment:
The `styleguide` can not be empty. `styleguide` is always set when calling this function (the function is typed like this :))